tools/migrations/rss_to_feed.sql

```diff
@@ -0,0 +1,15 @@
+/* Update the table to be feed rather than rss_feed.
+  Add the column feed_type which should match the handlers defined in 
+    zeeguu/core/feedhandler.py
+*/
+ALTER TABLE `zeeguu_test`.`rss_feed` RENAME TO  `zeeguu_test`.`feed`;
+ALTER TABLE `zeeguu_test`.`feed` ADD COLUMN `feed_type` INT NOT NULL DEFAULT 0 AFTER `deactivated`;
+
+/* Update the article table to match the updated feed table.
+*/
+ALTER TABLE `zeeguu_test`.`article` DROP FOREIGN KEY `article_ibfk_1`;
+ALTER TABLE `zeeguu_test`.`article` CHANGE COLUMN `rss_feed_id` `feed_id` INT NULL DEFAULT NULL ;
+ALTER TABLE `zeeguu_test`.`article` 
+ADD CONSTRAINT `article_ibfk_1`
```

Comment:
i wonder if there is a better naming of fk constraints. 
this conversation seems to suggest that article_fk_feed_id? https://dba.stackexchange.com/questions/15530/what-does-ibfk-stand-for-in-mysql
then again, this would only apply for the FKs that are defined by us. the ones generated seem to indeed of of the form ibfk_<number>

Reply:
I think that would mean updating all the FKs to a new naming scheme, as currently we always use [table]_ibfk_ . So, I would say that should be done as a different change?